api/v1/helpers/verbs/doGet.js

```diff
@@ -65,17 +68,18 @@ function doGet(req, res, next, props) {
       }
     });
   } else {
-    u.findByKey(props, req.swagger.params)
-    .then((o) => {
-      resultObj.dbTime = new Date() - resultObj.reqStartTime;
+    let getPromise;
+    if (featureToggles.isFeatureEnabled(constants.featureName) &&
+     props.modelName === 'Sample') {
+      getPromise = redisModelSample.getSample(req.swagger.params);
+    } else {
+      getPromise = u.findByKey(props, req.swagger.params);
+    }
 
-      // loop through remove values to delete property
-      if (props.fieldsToExclude) {
-        u.removeFieldsFromResponse(props.fieldsToExclude, o.dataValues);
-      }
+    getPromise.then((o) => {
+      resultObj.dbTime = new Date() - resultObj.reqStartTime;
 
-      // o is a sequelize obj, get dataValues obj
-      u.logAPI(req, resultObj, o.dataValues);
+      u.logAPI(req, resultObj, o);
```

Comment:
o is used by the logging utility to return the approximate the size of returned information. If o is the sequelize object or redis object, the approximation may be way off. 
We may need to modify the getSize(object) function in utils/apiLog.js to parse the object.